src/ir.cpp

```diff
@@ -12460,13 +12473,13 @@ static TypeTableEntry *ir_analyze_fn_call(IrAnalyze *ira, IrInstructionCall *cal
                         return ira->codegen->builtin_types.entry_invalid;
 
                     if (!ir_analyze_fn_call_generic_arg(ira, fn_proto_node, arg_tuple_arg, &impl_fn->child_scope,
-                        &next_proto_i, generic_id, &inst_fn_type_id, casted_args, impl_fn))
+                        &next_proto_i, generic_id, &inst_fn_type_id, &casted_args, impl_fn))
                     {
                         return ira->codegen->builtin_types.entry_invalid;
                     }
                 }
             } else if (!ir_analyze_fn_call_generic_arg(ira, fn_proto_node, arg, &impl_fn->child_scope,
-                &next_proto_i, generic_id, &inst_fn_type_id, casted_args, impl_fn))
+                &next_proto_i, generic_id, &inst_fn_type_id, &casted_args, impl_fn))
```

Comment:
Use pointer to function in case we have not allocated memory for the arguments. This can happen is number of arguments was determined to be zero upfront.